### PR TITLE
Add searchyt command

### PR DESCRIPTION
**Pull request by techsnap**
Adds searchyt command to bot

Things to be done:
- Multi word queries need to be handled
- Add new command to command list after bug fix

### CODE REVIEW

**Comment by z404** [comment]
@techsnap is this PR ready to be merged? It has been open for a while without any changes

**Comment by techsnap** [comment]
I will make final changes today.

**Comment by techsnap** [comment]
Hey, can you reopen this PR. I accidentally closed it. I have added searchyt functionality now.

**Comment by techsnap** [comment]
Sorry about that. Reopened it.

File: music.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
+
+const developerKey = ""
```
Comment (z404) [inline music.go:15]: Change this so that this variable is declared in `main.go` and the key is read from the `.env` file as `YOUTUBE_API_KEY`

**Review by z404 — CHANGES_REQUESTED** [review]
A small change is required, please take a look! Good work!

File: music.go (hunk 1 above)
Comment (techsnap) [reply music.go:15]: Working on it.

Comment (techsnap) [reply music.go]: Done. Found some other issues in music.go and fixed them as well.

**Review by z404 — APPROVED** [review]
LGTM! Good work :D

**Pull request merged**